src/components/notification/z-cookiebar/styles.css

```diff
@@ -0,0 +1,111 @@
+:host {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+}
+
+z-candybar.hidden {
+  display: none;
+}
+
+.content {
+  display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	align-items: stretch;
+  align-content: space-between;
+  color: var(--text-grey-800);
+  border-radius: var(--half-x1);
+  font-size: 16px;
+  letter-spacing: 0px;
+  line-height: 24px;
+  padding: var(--basex3) var(--basex2);
+}
+
+.content > div > h2 {
+  height: var(--half-x7);
+  margin: 0;
+  padding: 0;
+  color: rgb(75, 78, 82);
+  border-radius: 0px;
+  font-size: 18px;
+  font-weight: var(--font-sb);
+  letter-spacing: 0px;
+  line-height: 22px;
+  border-bottom: var(--border-base) solid rgb(75, 78, 82);
+}
+
+.content > div > div {
+  padding: var(--basex3) 0;
+  font-size: 14px;
+  letter-spacing: 0.16px;
+  line-height: 20px;
+}
+
+.content > div > div > p {
+  margin: 0;
+}
+
+.content > div > div > p > a {
+  cursor: pointer;
+  font-weight: var(--font-sb);
+  color: var(--text-grey-800);
+  text-decoration: underline;
+  height: 14px;
+  display: inline-flex;
+}
+
+.content > div > div > p > a:visited {
+  color: var(--text-grey-800);
+  cursor: pointer;
+  z-index: 1000;
+}
+
+.content > z-button {
+  margin: 0;
+  width: 100%;
+}
+
+/* Tablet breakpoint */
+@media only screen and (min-width: 768px) {
+
+  .content {
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+    font-size: 16px;
+    letter-spacing: 0px;
+    line-height: 24px;
+    padding: var(--basex3);
+  }
+
+  .content > div {
+    margin-right: 20%;
```

Comment:
Questa regola é stata validata con i designer?

Reply:
è il margine che c'è tra sezione col testo e z-button, da abstract è fissa, ma l'abstract non prevede mai visualizzazioni intermedie, da qui l'utilizzo di un margine percentuale. Se preferisci un modo diverso di rendere la cosa, o se pensi che sia scorretto posso cambiare

![image](https://user-images.githubusercontent.com/12939926/100133016-84984880-2e86-11eb-95a5-4852b3a3ed4c.png)

Reply:
In effetti é un vaso di Pandora, muoverei queste osservazioni ai designer, credo che il loro lavoro debba entrare anche nel merito di queste situazioni e questi dettagli.

Io per esempio userei l'unitá `ch` rendere la larghezza in relazione ai caratteri e quind favorire la lettura. Ci dovrebbe essere della letteratura che dice che righe troppo lunghe sono poco leggibili.

Peró questi pipponi non dobbiamo farceli noi al momento della codifica. O comunque se ce li facciamo, dobbiamo esportartli e smarcarli insieme a chi é davvero competente (ed io non lo sono, sia chiaro!).

Imho!

Reply:
mi sembra corretto, ne possiamo parlare alla call di giovedì o a quella congiunta di lunedì